src/app/reporting/ReportSchedulerImpl.cpp

```diff
@@ -18,6 +18,11 @@
 #include <app/InteractionModelEngine.h>
 #include <app/reporting/ReportSchedulerImpl.h>
 
+#ifndef ICD_REPORT_ON_ENTER_ACTIVE_MODE
```

Comment:
This change should not be needed if the build config is actually propagating to here correctly, right?

Reply:
right, this isn't needed anymore.